### PR TITLE
Add FOSSILIZE_REPLAY_WRAPPER env variable support.

### PR DESCRIPTION
**Pull request by d-riley**

### CODE REVIEW

**Comment by HansKristian-Work** [comment]
I'm not sure what problem this is intended to solve. Why not just invoke the wrapper directly?

**Comment by d-riley** [comment]
I don't have access to modify the caller (ie Steam Client), and @Plagman said to just make the change here.

**Review by HansKristian-Work — CHANGES_REQUESTED** [review]

File: cli/fossilize_replay.cpp [hunk 1]
```diff
@@ -3819,6 +3826,17 @@ int main(int argc, char *argv[])
 
 	bool log_memory = false;
 
+#ifndef _WIN32
+        // If a wrapper is specified, pass execution entirely to the wrapper.
```
[the remark this answers is not in the document]
Comment (moonfall) [reply cli/fossilize_replay.cpp:3829]: Done.

File: cli/fossilize_replay.cpp [hunk 2]
```diff
@@ -3819,6 +3826,17 @@ int main(int argc, char *argv[])
...
+        // Wrapper is responsible for clearing the environment to prevent loops.
+        const char *wrapper_path = getenv(FOSSILIZE_REPLAY_WRAPPER_ENV);
+        if (wrapper_path && *wrapper_path) {
```
[the remark this answers is not in the document]
Comment (moonfall) [reply cli/fossilize_replay.cpp]: Done.

File: cli/fossilize_replay.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (d-riley) [reply cli/fossilize_replay.cpp:3829]: Done. (Sorry for dup comments.)

File: cli/fossilize_replay.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (d-riley) [reply cli/fossilize_replay.cpp]: Done.

**Pull request merged**